src/websock/native/option.go

```diff
@@ -0,0 +1,21 @@
+package native
+
+import "time"
+
+// Option is used to configure the Handler
+type Option interface {
+	modify(*visitor)
+}
+
+// MaxSyncCallTimeout sets the maximum time a call can be
```

Comment:
- Don't forget punctuation, Go docs are complete sentences.
- Outside of the framing, Rinq does not use the term "sync call" in the public API, so maybe just `MaxCallTimeout`
- Docs should explain that it overrides a timeout sent by the client *and* that it doesn't affect async calls.